pkg/network/ebpf/c/protocols/events.h

```diff
@@ -91,6 +91,21 @@
         if (name##_batch_full(batch)) {                                                 \
             batch_state->idx++;                                                         \
         }                                                                               \
+    }                                                                                   \
+                                                                                        \
+    static __always_inline void name##_batch_pop() {                                    \
```

Comment:
can you add documentation over the function?

Reply:
sure!